src/sgame/sg_buildable.cpp

```diff
@@ -2384,6 +2384,7 @@ void G_LayoutSelect()
 		else
 		{
 			//use the map's builtin layout
+			Cvar::SetValue( "layout", "" );
```

Comment:
The function has another early return, where it says `"None of the specified layouts could be found, using map default."`

Reply:
Forgot to commit it :(